packages/@glimmer/opcode-compiler/lib/opcode-builder/helpers/components.ts

```diff
@@ -221,6 +222,10 @@ export function InvokeStaticComponent({
     // to blocks (they aren't accessible from userland anyways), so we push an
     // empty array instead of the actual block names.
     out.push(op(Op.PushArgs, strArray(names), strArray(EMPTY_ARRAY), flags));
+
+    // And push an extra pop to pop the args off before we begin setting
```

Comment:
> push an extra pop to pop

Is this phrasing right? It reads a bit confusingly

Reply:
Can rephrase, "push an extra pop operation to remove the args"